docs/source/reference-lowlevel.rst

```diff
@@ -711,7 +711,8 @@ can use as a model:
 
 .. code:: python
 
-    import asyncio, trio
+    import asyncio
+    import trio
```

Comment:
Don't know why this raised errors, or by what, but it gave
```
/home/h/Git/trio/docs/source/reference-lowlevel.rst: WARNING: Could not match transformation of `asyncio` on source lines 1-1
```

Reply:
Let's open an upstream issue for that?  (I assume we can get a reproducer for codeautolink)